src/documentation/examples/Loader/Loader.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Loader from '@bufferapp/ui/Loader';
+
+/** Loader Example */
+export default function ExampleLoader() {
+  return (
+    <Loader>
+      Hello, world!
```

Comment:
I don't think we are using the children text 😄

Reply:
good catch!